### ISSUE TITLE
Unable to see GINI and GDP/cap description and source

### ISSUE THREAD

**Issue by jheeffer** [vizabi/mountainchart#11]
There is currently no way for users to see the description and source of the GINI and GDP/cap data used to draw the mountain chart.

http://www.gapminder.org/tools/#_locale_id=en;&chart-type=mountain

A user asked for this through an email. Seems right that it should be added somewhere?

**Comment by angiehjort** [comment]
@jheeffer Yes, we should put question mark near the title and link it to the blog post. In the meantime pls send the user the link you can find inside "data doubts"
